pkg/models/org.go

```diff
@@ -10,11 +10,12 @@ import (
 
 // OrgMetadata has the information about the organziation.
 type OrgMetadata struct {
-	OrgID            uuid.UUID              `json:"org_id"`
-	Name             string                 `json:"org_name"`
-	MaxUsers         *int                   `json:"max_users"`
-	Metadata         map[string]interface{} `json:"metadata"`
-	IsSamlConfigured bool                   `json:"is_saml_configured"`
+	OrgID                 uuid.UUID              `json:"org_id"`
+	Name                  string                 `json:"org_name"`
+	MaxUsers              *int                   `json:"max_users"`
+	Metadata              map[string]interface{} `json:"metadata"`
+	IsSamlConfigured      bool                   `json:"is_saml_configured"`
+	CustomRoleMappingName *string                `json:"custom_role_mapping_name"`
```

Comment:
I think this might mean it's not backwards compatible, so existing user's code will break for this -- should be double checked